content/en/synthetics/browser_tests/actions.md

```diff
@@ -13,106 +15,190 @@ further_reading:
+Assertions allow you to validate that your browser test is in the state you expect it to be in at any point of a simulated user journey. 
+
+To confirm your test ends in an expected state, you must end your browser tests with an **Assertion**.
+
+{{< img src="synthetics/browser_tests/browser_test_assertions.png" alt="Options for assertions in a browser test step" style="width:70%;" >}}
 
-Assertions allow you to validate that your browser test is in the state you expect it to be in, at any given point of a simulated user journey. This is why you must end your browser tests by an **Assertion** to confirm it ended up in an expected state.
+Some assertions are performed on the active page, which refers to the page that the user last interacted with such as a **Click** or an **Assertion** on a page element.
 
-Some assertions are performed on the **active page**. Active page refers to the page that has experienced the last interaction like using a **Click** or an **Assertion** on a given page element for instance.
+To create an assertion step, select an assertion type:
+
+{{< tabs >}}
+{{% tab "Test An Element On The Active Page" %}}
 
 #### Test an element's content
 
-Selects an element and checks if it contains a specific value. For instance, you could select a `div` and check whether it contains the word "hello".
+Create this assertion step to have your browser test select a page element and check if it contains a specific value. 
+
+You can specify that the content `should equal`, `should not equal`, `should contain`, `should not contain`, `should start with`, `should not start with`, `should match regex`, `should be empty`, `should not be empty`, `should be greater than`, `should be greater than or equal`, `should be less than`, or `should be less than or equal` a given value.
+
+To save the assertion, click **Apply**. You can select a `div` that `should contain` the word `hello` as a given value.
 
 #### Test an element's attribute
 
-Selects an element of your page and checks if one of its attributes matches the expected content. For instance, you could test that an `src` attribute value matches the path of the expected image.
+Create this assertion step to have your browser test select a page element and check if one of its attributes matches the expected content. 
 
-#### Test that some text is present on the active page
+Enter a name for the page attribute. You can specify that the content `should equal`, `should not equal`, `should contain`, `should not contain`, `should start with`, `should not start with`, `should match regex`, `should be empty`, `should not be empty`, `should be greater than`, `should be greater than or equal`, `should be less than`, or `should be less than or equal` a given value.
+
+To save the assertion, click **Apply**. You can test that a `src` attribute value matches the path of the expected image.
 
-Asserts that some specific text is present on the current page.
+#### Test that an element is present
```

Comment:
add something like in this section
we recommend you to use the 2 previous assertions that by their nature are more accurate